modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/ZonePartitionReplicaListener.java

```diff
@@ -53,26 +76,51 @@ public class ZonePartitionReplicaListener implements ReplicaListener {
     // TODO: https://issues.apache.org/jira/browse/IGNITE-22624 await for the table replica listener if needed.
     private final Map<TablePartitionId, ReplicaListener> replicas = new ConcurrentHashMap<>();
 
-    private final RaftCommandRunner raftClient;
+    private final PlacementDriver placementDriver;
+
+    /** Instance of the local node. */
+    private final ClusterNode localNode;
+
+    /** Clock service. */
+    private final ClockService clockService;
+
+    private final RaftCommandRunner raftCommandRunner;
+
+    private final ZonePartitionId zoneReplicationGroupId;
 
     private final TxFinishReplicaRequestHandler txFinishReplicaRequestHandler;
 
     /**
      * The constructor.
      *
+     * @param txStatePartitionStorage Storage for transactions' states.
```

Comment:
I would prefer to remove all these `@param` tags instead. They serve no purpose and are very cumbersome to support in actual state.